tests/TestHelpers/GraphHelper.php

```diff
@@ -1426,4 +1426,45 @@ public static function unassignRole(
 			self::getRequestHeaders(),
 		);
 	}
+
+	/**
+	 * @param string $baseUrl
+	 * @param string $xRequestId
+	 * @param string $user
+	 * @param string $password
+	 * @param string $path
+	 *
+	 * @return string
+	 * @throws GuzzleException
+	 * @throws Exception
+	 */
+	public static function getMountPointId(
+		string $baseUrl,
+		string $xRequestId,
+		string $user,
+		string $password,
+		string $path
+	): string {
+		$response = self::getMySpaces(
+			$baseUrl,
+			$user,
+			$password,
+			'',
+			$xRequestId
+		);
+		$drives = json_decode($response->getBody()->getContents(), true, 512, JSON_THROW_ON_ERROR);
+
+		// the response returns the shared resource in driveAlias all in lowercase,
+		// For example: if we get the property of a shared resource "FOLDER" then the response contains "driveAlias": "mountpoint/folder"
```

Comment:
what will be in the response if we have two shares having same name?

Reply:
In case of two shares with the same name, the response for the second shared resource, for e.g. `folder` contains `"driveAlias": "mountpoint/folder-(2)"`.
Actual Response:
```json
"driveAlias": "mountpoint/folder-(2)",
"driveType": "mountpoint",
"id": "a0ca6a90-a365-4782-871e-d44447bbc668$a0ca6a90-a365-4782-871e-d44447bbc668!1fb0c692-42cc-4884-a578-9a69d79a7701:f7fbf8c8-139b-4376-b307-cf0a8c2d0d9c:b808b6aa-5bf5-4de7-8512-568851141767",
"name": "folder (2)",
"owner": {
  "user": {
    "displayName": "",
    "id": "f7fbf8c8-139b-4376-b307-cf0a8c2d0d9c"
  }
}
```

Reply:
please, add it to the comments in the code